### DISCUSSION TITLE
[Small proposal / Feature request] Add a "download counter" to https://rubygems.org/pages/download. a simple text message (simplest to implement probably)

### DISCUSSION THREAD

**Discussion by rubyFeedback** [rubygems/rubygems.org#3061] [General]
This small proposal is a feature request concerning the following URL:

    https://rubygems.org/pages/download

It is proposed herein that a "statistics" information is added to that page.
With that I mean a "tell users how many downloads occurred per gem
release", roughly.

It does not have to be 100% accurate, but may group it into 
"n thousands" downloads aka "560.000 downloads approximately"
or something like that.

For instance, for the current gem version v3.3.13, that page could
either have a new sub-link, such as:

    https://rubygems.org/pages/download/stats
    https://rubygems.org/pages/download/statistics # or this variant

Or, if that is too much work, then perhaps simply display it on
the bottom, such as:

"There have been about 503.000 downloads of gem version 3.3.13
so far."

Or something like that.

Rationale: this is mostly just a short visual information for the
user. It may also help to see which gem versions are more popular
than others. Perhaps weekly download counter may be even better
BUT I wanted to keep this proposal simple, so to distill it in
its simplicity into a TL;DR variant, it can be summarized as
follows:

"Consider adding a small download-counter number per gem version
for the webpage at https://rubygems.org/pages/download/."

**Comment by simi** [comment]
If I understand it well, you can find this number at https://rubygems.org/gems/rubygems-update (since that's the package behind update).

**Comment by sonalkr132** [comment]
rubygem-update is one of the sources of rubygems download. Ideally we would want to know users per version.
> It may also help to see which gem versions are more popular
than others. 

related usecase is being discussed here https://github.com/rubygems/rubygems.org/issues/1335 please leave your comments on that thread. You can see approx numbers here as well https://ecosystem.rubytogether.org/
